### DISCUSSION TITLE
Mympd no web interface.

### DISCUSSION THREAD

**Discussion by Francy2007** [community/community#14750] [Projects and Issues]
Hello,
I have installed Mympd 9.2.3-1 amd64 and Mpd 0.21.20 on the same pc, with Ubuntu 20.04, but I have no web interface connecting with localhost: 80.
The configuration in /etc/mympd.conf is as follows:

 # Consult documentation at https://jcorporation.github.io/myMPD/configuration
config mympd main
        # CAUTION: there can be a lot of writes to workdir directory
        # and if it's on raw flash, this can kill it very fast.
        option workdir '/ var / lib / mympd'
        option mpd_host 'localhost'
        option mpd_port '6600'
        option acl ''
        option http_host 'localhost'
        option http_port '80'
        option loglevel '5'

        # Requires enabled MYMPD_LUA at compilation time
        option lualibs 'all'
        option scriptacl '+127.0.0.1'

        # Requires enabled MYMPD_SSL at compilation time
        option ssl 'true'
        option ssl_port '443'
        option ssl_san ''
               
If I give the command "sudo systemctl status mympd" I have the following answer:

 
● mympd.service - myMPD server daemon
     Loaded: loaded (/lib/systemd/system/mympd.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Tue 2022-04-12 06:07:55 CEST; 21min ago
    Process: 1094 ExecStart=/usr/bin/mympd (code=exited, status=1/FAILURE)
   Main PID: 1094 (code=exited, status=1/FAILURE)

apr 12 06:07:54 vincenzo-imedia-S3800 systemd[1]: Started myMPD server daemon.
apr 12 06:07:54 vincenzo-imedia-S3800 mympd[1094]: NOTICE   mympd     Work dir: "/var/lib/mympd"
apr 12 06:07:54 vincenzo-imedia-S3800 mympd[1094]: NOTICE   mympd     Config dir: "/var/lib/mympd/config"
apr 12 06:07:54 vincenzo-imedia-S3800 mympd[1094]: NOTICE   mympd     Cache dir: "/var/cache/mympd"
apr 12 06:07:55 vincenzo-imedia-S3800 mympd[1094]: NOTICE   mympd     Setting loglevel to NOTICE
apr 12 06:07:55 vincenzo-imedia-S3800 mympd[1094]: NOTICE   mympd     Starting myMPD 9.2.3
apr 12 06:07:55 vincenzo-imedia-S3800 mympd[1094]: EMERG    mympd     Can't bind to http://0.0.0.0:80
apr 12 06:07:55 vincenzo-imedia-S3800 systemd[1]: mympd.service: Main process exited, code=exited, status=1/FAILURE
apr 12 06:07:55 vincenzo-imedia-S3800 systemd[1]: mympd.service: Failed with result 'exit-code'.
~

How could I solve?

Vincenzo, Italy

**Comment by jcorporation** [comment]
You should ask this here: https://github.com/jcorporation/myMPD/discussions
